pymc/sampling.py

```diff
@@ -1746,13 +1745,16 @@ def sample_posterior_predictive(
     if size is not None:
         vars_to_sample = [change_rv_size(v, size, expand=True) for v in vars_to_sample]
 
+    if compile_kwargs is None:
```

Comment:
Seems like the tests didn't catch that.

Reply:
What didn't they catch? There were a bunch of failing tests. Is there an error still?

Reply:
I didn't check, but did the test fail before you fixed the wrong name or did you add a new one that would have caught it?